tests/e2e/ctl_v3_role_test.go

```diff
@@ -53,6 +54,40 @@ func roleAddTest(cx ctlCtx) {
 	}
 }
 
+func rootRoleGetTest(cx ctlCtx) {
+	cmdSet := []struct {
+		args        []string
+		expectedStr string
+	}{
+		// Add a role of root .
+		{
+			args:        []string{"add", "root"},
+			expectedStr: "Role root created",
+		},
+		// get root role.
+		{
+			args:        []string{"get", "root"},
+			expectedStr: "Role root\nKV Read:\t[, <open ended>\nKV Write:\t[, <open ended>",
+		},
+		{
```

Comment:
Please add a comment that behavior of the commands bellow is subject to change as:
  - granting to root should be refused by server
  - get root role should always return `[, <open ended>`

Reply:
done, thanks for your detailed guidance.